doc/UseCases/uc-gconf_x-road_use_case_model_for_global_configuration_distribution_1.4_Y-883-8.md

```diff
@@ -0,0 +1,1908 @@
+    Federation. Document ID: UC-FED.
+
+9.  <a id="Ref_UG-SYSPAR" class="anchor"></a>\[UG-SYSPAR\] X-Road:
+    System Parameters. Document ID: UG-SYSPAR.
+
+## 2 Overview
+
+The X-Road security servers periodically download global configuration
+distributed by the configuration providers. The global configuration is
+used to verify the parties communicating over the X-Road and to check
+the validity of various data items, such as authentication certificates,
+OCSP responses and timestamps.
+
+The information needed by the security servers for downloading and
+verifying global configuration is contained in configuration anchors.
+The configuration anchors are distributed by the internal configuration
+providers to the security server owners via out of band means.
+
+The configuration providers ensure the integrity of the distributed
+configuration by signing the configuration directory.
+
+### 2.1 Actors
+
+The use case model for downloading configuration includes the following
+actors.
+
+-   **SS administrator** (security server administrator) – a person
+    responsible for managing the security server.
+
+-   **CS administrator** (central server administrator) – a person
+    responsible for managing the central server.
+
+-   **Configuration source** – a component (HTTP server) managed by the
+    central server or the configuration proxy that distributes global
+    configuration.
+
+-   **Configuration client** – security server that downloads
+    configuration from configuration source(s).
+
+Relationships between the actors, systems and use cases are described in
+Figure 1.
+
+![](img/use_case_diagaram_disto_global_conf.PNG)
+
+Figure 1. Use case diagram for distributing global configuration
+
+### 2.2 Central Server Use Cases
+
+#### 2.2.1 UC GCONF\_01: View a Configuration Source 
+
+**System**: Central server
+
+**Level**: User task
+
+**Component:** Central server
+
+**Actor**: CS administrator
+
+**Brief Description**: CS administrator views the information about a
+configuration source provided by the central server.
+
+**Preconditions**: -
+
+**Postconditions**: The configuration source information has been
+displayed to CS administrator.
+
+**Trigger**: CS administrator wishes to view the configuration source
+information.
+
+**Main Success Scenario**:
+
+1.  CS administrator selects to view a configuration source.
+
+2.  System displays a configuration source provided by the central
+    server. The following information is displayed.
+
+    -   Type of the configuration source (internal/external).
+
+    -   The SHA-224 hash value of the configuration anchor.
+
+    -   The generation date and time (UTC) of the configuration anchor.
+
+    -   The configuration download URL – address from where the
+        configuration directory provided by this source can be
+        downloaded. The system composes the download URL by adding
+        */internalconf* or []{#conf_url35 .anchor}*/externalconf*
```

Comment:
"[]{#conf_url35 .anchor}" is unnecessary here.

Reply:
fixed